doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-% Author: Pavol Loffay
-% Project: Master thesis - Hawkular alert prediction
-% Date 20.2.2015
-
-% Cross-Referencing Conventions:
-% chap: chapter
-% sec: section
-%
-% appen: everything from appendix, e.g. appen:img:foo
-%
-% eq: equation
-% img: image
-% tab: table
-% alg: algorithms
-% item: item in itemize
-%
-% example \label{img:foo-foo}
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Introduction} \label{chap:introduction}
-It is important to assure an application health and reliability when driving successful business on the internet. One
-can achieve that by monitoring subjected resources and setting up clever alerting system.
-
```

Comment:
When driving successful business on the internet, it is important to assure application`s health and reliability.